ansible/roles/products/common/resources/templates/scripts/start-docker-container-local.j2

```diff
@@ -7,7 +7,12 @@ NC='\033[0m' # No Color
 if {{ upgrade_check | lower }}; then
     "{{ user_folder }}/.shadow_launcher_app/shadow_hand_launcher/{{ upgrade_script }}"
 fi
+if [ "$(docker ps -q -f name={{ container_name  }})" ]; then
```

Comment:
is this part of this pr?

Reply:
no. I removed it now. Fixed in the last commit